pkg/go/transformer/module-to-model.go

```diff
@@ -81,7 +81,7 @@ func TransformModuleFilesToModel( //nolint:funlen,gocognit,cyclop
 
 		mdl, typeDefExtensions, err := TransformModularDSLToProto(module.Contents)
 		if err != nil {
-			var syntaxError *OpenFgaDslSyntaxMultipleError
+			var syntaxError *multierror.Error
```

Comment:
Looks good to me, however pinging @d-jeffery as this change may break some stuff in VS Code handling

Reply:
I think on the JS side there shouldn't be any change as we were correctly handling them already, however because I had the type check wrong in Go syntax errors were being ignored so a transform was appearing successful and just dropping files (you don't want to know how long I wasted trying to figure out why some relations were missing in my CLI testing 😭 )